packages/create-vite/README.md

```diff
@@ -29,6 +29,12 @@ With Bun:
 bun create vite
 ```
 
+With Deno:
+
+```bash
+deno run -A npm:create-vite
```

Comment:
I think we should use the same command with https://github.com/vitejs/vite/blob/9f2247c066cac75746356c9391845235445a154b/docs/guide/index.md?plain=1#L65

Also I think we should add deno in the code block below.